api/swagger-spec/v1.json

```diff
@@ -11228,6 +11228,13 @@
       },
       "description": "IP addresses which offer the related ports."
```

Comment:
Are addresses considered ready if in the "addresses" array ?
Or are "addresses" the collection of both ready and notReady addresses ?

Is is possible for an address to ready if it is ready on all ports ?

Can the description answer these questions explicitly ?